queries.sql

```diff
@@ -1,3 +1,41 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM animals WHERE name LIKE '%mon';
+SELECT * FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-31';
+SELECT * FROM animals WHERE neutered = TRUE AND escape_attempts < 3;
+SELECT date_of_birth FROM animals WHERE name = 'Agumon' OR name = 'Pikachu';
+SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
+SELECT * FROM animals WHERE neutered = TRUE;
+SELECT * FROM animals WHERE name != 'Gabumon';
+SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
+
+-- Update data
+BEGIN;
+UPDATE animals SET species='unspecified';
+SELECT * FROM animals;
+ROLLBACK;
+SELECT * FROM animals;
+
+UPDATE animals SET species='digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species='pokemon' WHERE species IS NULL;
+SELECT * FROM animals;
+COMMIT;
+
```

Comment:
- [ ] After `COMMIT` please verify that the change persists after the commit

Reply:
- [ ] Please add queries that are missing
1. Inside a transaction delete all records in the animal's table, then roll back the transaction
2. How many animals are there?
3. How many animals have never tried to escape?
4. What is the average weight of animals?
5. Who escapes the most, neutered or not neutered animals?
6. What is the minimum and maximum weight of each type of animal?
7. What is the average number of escape attempts per animal type of those born between 1990 and 2000?

Reply:
- [ ] Please remember to add a screenshots of your queries results to the pull request description

Reply:
I added the screenshot in the description. To make them easier to identify, I will add them here as well: 
![Screenshot 2023-09-27 9 34 15 AM](https://github.com/wacoo/vet-clinic-database/assets/9458141/f76089de-a646-45cc-9116-b535685e9ddc)
![Screenshot 2023-09-27 9 52 49 AM](https://github.com/wacoo/vet-clinic-database/assets/9458141/b11af82a-fcd2-43bf-9802-ee325659b120)
![Screenshot 2023-09-27 11 34 39 AM](https://github.com/wacoo/vet-clinic-database/assets/9458141/7bdc01a5-fe2e-4f1d-adac-be81b3d7a33e)